packages/scikit-image/meta.yaml

```diff
@@ -1,6 +1,9 @@
 package:
   name: scikit-image
   version: 0.19.3
+  top-level:
+    - skimage
+    # - doc  # scikit-image has top level `doc` directory... but it is too common name so we don't want to use it as a hint for top-level import.
```

Comment:
scikit-image have top-level doc directory, but the name doc is too common so I think we don't want to load `scikit-image` is someone do `import doc`.

Reply:
Reported to https://github.com/scikit-image/scikit-image/issues/6505